src/v/kafka/server/group.cc

```diff
@@ -47,6 +47,11 @@ namespace kafka {
 
 using member_config = join_group_response_member;
 
+static const model::violation_recovery_policy crash{
+  model::violation_recovery_policy::crash};
+static const model::violation_recovery_policy best_effort{
+  model::violation_recovery_policy::best_effort};
```

Comment:
maybe try out this new c++20 feature: https://oleksandrkvl.github.io/2021/04/02/cpp-20-overview.html#using-enum

Reply:
doesn't work:

> [rp:build] ../../../src/v/kafka/server/group.cc:1214:11: error: expected unqualified-id
> [rp:build]     using enum model::violation_recovery_policy;

Reply:
I've made it nicer with `using violation_recovery_policy = model::violation_recovery_policy;` instead

Reply:
I think you need to include the full enum value like `using enum model::violation_recovery_policy::best_effort`

Reply:
but no need to change anything